.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.html]
+indent_size = 4
```

Comment:
I'm a 2 space for pretty much anything buy python. Any chance you could switch `.html` back to 2?

I don't think this will conflict with prettier as prettier sort of has the final say when it comes to JS code since the git commit runs the command pre push. 

Also, I think adding something like this for `elm` would work:

```
[*.elm]
indent_style = space
indent_size = 4
```